### PR TITLE
Update stripe endpoint

### PR DESCRIPTION
**Pull request by tomverran**
Push CC updates straight through to Zuora

### CODE REVIEW

File: membership-attribute-service/conf/DEV.conf [hunk 1]
```diff
@@ -4,7 +4,6 @@ include "application.conf"
 
 identity {
   production.keys = false
-  test.users.secret="a-non-secure-key-for-our-dev-env-only"
```
Comment (tomverran) [inline membership-attribute-service/conf/DEV.conf]: this was removed as it breaks test users on dev which is pretty annoying

File: membership-attribute-service/app/configuration/Config.scala [hunk 2]
```diff
@@ -49,6 +49,13 @@ object Config {
   val mmaCorsConfig = CORSConfig.denyAll.copy(
     allowedOrigins = config.getStringList("mma.cors.allowedOrigins").asScala.toSet
   )
+
+  lazy val mmaCardCorsConfig = Config.mmaCorsConfig.copy(
+    isHttpHeaderAllowed = Seq("accept", "content-type", "csrf-token").contains(_),
```
Comment (afiore) [inline membership-attribute-service/app/configuration/Config.scala]: Shoudn't these headers be capitalised?

File: membership-attribute-service/app/controllers/AttributeController.scala [hunk 3]
```diff
@@ -49,6 +55,25 @@ class AttributeController {
...
+    val updateForm = Form { single("stripeToken" -> nonEmptyText) }
+    val productFamily = request.touchpoint.ratePlanIds(product)
+    val tp = request.touchpoint
+
+    (for {
+      user <- OptionT(Future.successful(authenticationService.userId))
+      sfUser <- OptionT(tp.contactRepo.get(user))
+      subscription <- OptionT(tp.subService.findByProductFamily(sfUser, productFamily))
+      stripeCardToken <- OptionT(Future.successful(updateForm.bindFromRequest().value))
+      result <- OptionT(tp.subService.setPaymentCardWithStripeToken(subscription.accountId, stripeCardToken))
```
Comment (afiore) [inline membership-attribute-service/app/controllers/AttributeController.scala]: result is unbound... can it be replaced with `_`

File: membership-attribute-service/app/configuration/Config.scala (hunk 2 above)
Comment (tomverran) [inline membership-attribute-service/app/configuration/Config.scala]: turns out the cors filter does some normalisation after all so the case is correct

**Comment by afiore** [comment]
:+1: looks good

**Pull request merged**